src/main/java/com/marmelab/quoridor/web/BoardView.java

```diff
@@ -38,6 +40,40 @@ public BoardView(final Game game) {
 
         addHorizontalFences = availableNewFence.getAddHorizontalFences();
         addVerticalFences = availableNewFence.getAddVerticalFences();
+
+        possibleMoves = buildPossibleMoves(game.getBoard().getBoardSize());
+        squares.removeAll(possibleMoves.entrySet());
+    }
+
+    private Map<Position, CardinalDirection> buildPossibleMoves(final int boardSize) {
```

Comment:
Why is it build here and get everywhere else? What does the word difference mean to you?